client/index.js

```diff
@@ -152,14 +151,8 @@ addFilter(
 			capability: 'manage_woocommerce',
 		} );
 
-		// If disputes on transaction page feature is enabled, set up a soft
-		// redirect component; otherwise register the (legacy) dispute details page.
-		const isDisputeOnTransactionPageEnabled =
-			window.wcpaySettings.featureFlags.isDisputeOnTransactionPageEnabled;
 		pages.push( {
-			container: isDisputeOnTransactionPageEnabled
-				? RedirectToTransactionDetails
-				: LegacyDisputeDetailsPage,
+			container: RedirectToTransactionDetails,
```

Comment:
We can also remove `LegacyDisputeDetailsPage` completely.

Reply:
I believe this is being handled in #7363 and is OK to merge after 6.6.